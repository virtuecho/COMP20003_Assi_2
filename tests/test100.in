Windsor
Ascot Vale
Parkville
Kensington
Melbourne
Richmond
Ascot
Red Hill
Brunswick
Kensington Gardens
Kensington Park
Parkvile
