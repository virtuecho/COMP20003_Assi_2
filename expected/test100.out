Windsor -->
COMP20003 Code1054, Official Code Suburb41615, Official Name SuburbWindsor, Year2021, Official Code State4, Official Name StateSouth Australia, Official Code Local Government Area40150, Official Name Local Government AreaAdelaide Plains, Latitude-34.4098434, Longitude138.3147290
COMP20003 Code4338, Official Code Suburb33096, Official Name SuburbWindsor, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area31000, Official Name Local Government AreaBrisbane, Latitude-27.4343963, Longitude153.0301686
COMP20003 Code7057, Official Code Suburb14343, Official Name SuburbWindsor, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area13800, Official Name Local Government AreaHawkesbury, Latitude-33.6090236, Longitude150.8211213
COMP20003 Code8972, Official Code Suburb22805, Official Name SuburbWindsor, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, 25900, 26350, Official Name Local Government AreaMelbourne, Port Phillip, Stonnington, Latitude-37.8546754, Longitude144.9932064
Ascot Vale -->
COMP20003 Code960, Official Code Suburb20075, Official Name SuburbAscot Vale, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area25060, Official Name Local Government AreaMoonee Valley, Latitude-37.7770335, Longitude144.9136945
Parkville -->
COMP20003 Code10331, Official Code Suburb22038, Official Name SuburbParkville, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, 25060, 25250, Official Name Local Government AreaMelbourne, Moonee Valley, Moreland, Latitude-37.7866675, Longitude144.9512266
COMP20003 Code15232, Official Code Suburb13165, Official Name SuburbParkville, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area17620, Official Name Local Government AreaUpper Hunter Shire, Latitude-31.9705195, Longitude150.8823423
Kensington -->
COMP20003 Code1117, Official Code Suburb21327, Official Name SuburbKensington, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24330, 24600, 25060, Official Name Local Government AreaMaribyrnong, Melbourne, Moonee Valley, Latitude-37.7943466, Longitude144.9270189
COMP20003 Code1986, Official Code Suburb31507, Official Name SuburbKensington, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area31820, Official Name Local Government AreaBundaberg, Latitude-24.9170988, Longitude152.3182806
COMP20003 Code6206, Official Code Suburb50752, Official Name SuburbKensington, Year2021, Official Code State5, Official Name StateWestern Australia, Official Code Local Government Area57840, 58510, Official Name Local Government AreaSouth Perth, Victoria Park, Latitude-31.9856189, Longitude115.8839737
COMP20003 Code7720, Official Code Suburb12107, Official Name SuburbKensington, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area16550, Official Name Local Government AreaRandwick, Latitude-33.9134157, Longitude151.2207450
COMP20003 Code10486, Official Code Suburb40677, Official Name SuburbKensington, Year2021, Official Code State4, Official Name StateSouth Australia, Official Code Local Government Area45290, Official Name Local Government AreaNorwood Payneham and St Peters, Latitude-34.9237458, Longitude138.6456183
Melbourne -->
COMP20003 Code6623, Official Code Suburb21640, Official Name SuburbMelbourne, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area24600, 25900, 26350, 27350, Official Name Local Government AreaMelbourne, Port Phillip, Stonnington, Yarra, Latitude-37.8249613, Longitude144.9715278
Richmond -->
COMP20003 Code577, Official Code Suburb41256, Official Name SuburbRichmond, Year2021, Official Code State4, Official Name StateSouth Australia, Official Code Local Government Area48410, Official Name Local Government AreaWest Torrens, Latitude-34.9386376, Longitude138.5621269
COMP20003 Code2856, Official Code Suburb60551, Official Name SuburbRichmond, Year2021, Official Code State6, Official Name StateTasmania, Official Code Local Government Area61410, Official Name Local Government AreaClarence, Latitude-42.7387184, Longitude147.4137797
COMP20003 Code8552, Official Code Suburb32424, Official Name SuburbRichmond, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area34770, Official Name Local Government AreaMackay, Latitude-21.0869211, Longitude149.1405356
COMP20003 Code12417, Official Code Suburb13375, Official Name SuburbRichmond, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area13800, Official Name Local Government AreaHawkesbury, Latitude-33.6048433, Longitude150.7574251
COMP20003 Code12852, Official Code Suburb32425, Official Name SuburbRichmond, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area36300, Official Name Local Government AreaRichmond, Latitude-20.7622455, Longitude143.2365906
COMP20003 Code14926, Official Code Suburb22170, Official Name SuburbRichmond, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area27350, Official Name Local Government AreaYarra, Latitude-37.8202671, Longitude145.0024290
Ascot -->
COMP20003 Code5147, Official Code Suburb50031, Official Name SuburbAscot, Year2021, Official Code State5, Official Name StateWestern Australia, Official Code Local Government Area50490, Official Name Local Government AreaBelmont, Latitude-31.9340902, Longitude115.9294786
COMP20003 Code8205, Official Code Suburb30089, Official Name SuburbAscot, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area31000, Official Name Local Government AreaBrisbane, Latitude-27.4298286, Longitude153.0652855
COMP20003 Code9272, Official Code Suburb20073, Official Name SuburbAscot, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area20570, Official Name Local Government AreaBallarat, Latitude-37.3964624, Longitude143.8130771
COMP20003 Code12570, Official Code Suburb30090, Official Name SuburbAscot, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area36910, Official Name Local Government AreaToowoomba, Latitude-27.8254088, Longitude151.9968069
COMP20003 Code14412, Official Code Suburb20074, Official Name SuburbAscot, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area22620, Official Name Local Government AreaGreater Bendigo, Latitude-36.7091600, Longitude144.3352295
Red Hill -->
COMP20003 Code4282, Official Code Suburb13343, Official Name SuburbRed Hill, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area17080, Official Name Local Government AreaSnowy Valleys, Latitude-35.1584444, Longitude148.3723354
COMP20003 Code4794, Official Code Suburb51285, Official Name SuburbRed Hill, Year2021, Official Code State5, Official Name StateWestern Australia, Official Code Local Government Area58050, Official Name Local Government AreaSwan, Latitude-31.8275678, Longitude116.0745512
COMP20003 Code5926, Official Code Suburb13344, Official Name SuburbRed Hill, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area17310, Official Name Local Government AreaTamworth Regional, Latitude-30.4937039, Longitude150.6341084
COMP20003 Code7035, Official Code Suburb32401, Official Name SuburbRed Hill, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area37310, Official Name Local Government AreaWestern Downs, Latitude-26.6439922, Longitude150.6450088
COMP20003 Code7903, Official Code Suburb80112, Official Name SuburbRed Hill, Year2021, Official Code State8, Official Name StateAustralian Capital Territory, Official Code Local Government Area89399, Official Name Local Government AreaUnincorporated ACT, Latitude-35.3334057, Longitude149.1206203
COMP20003 Code8540, Official Code Suburb32400, Official Name SuburbRed Hill, Year2021, Official Code State3, Official Name StateQueensland, Official Code Local Government Area31000, Official Name Local Government AreaBrisbane, Latitude-27.4525092, Longitude153.0025949
COMP20003 Code9130, Official Code Suburb22148, Official Name SuburbRed Hill, Year2021, Official Code State2, Official Name StateVictoria, Official Code Local Government Area25340, Official Name Local Government AreaMornington Peninsula, Latitude-38.3635428, Longitude145.0160941
COMP20003 Code12786, Official Code Suburb13345, Official Name SuburbRed Hill, Year2021, Official Code State1, Official Name StateNew South Wales, Official Code Local Government Area17950, Official Name Local Government AreaWarren, Latitude-31.7345343, Longitude147.7943696
Brunswick -->
Kensington Gardens -->
COMP20003 Code4898, Official Code Suburb40678, Official Name SuburbKensington Gardens, Year2021, Official Code State4, Official Name StateSouth Australia, Official Code Local Government Area40700, Official Name Local Government AreaBurnside, Latitude-34.9197249, Longitude138.6631866
Kensington Park -->
COMP20003 Code11537, Official Code Suburb40679, Official Name SuburbKensington Park, Year2021, Official Code State4, Official Name StateSouth Australia, Official Code Local Government Area40700, Official Name Local Government AreaBurnside, Latitude-34.9201888, Longitude138.6542865
Parkvile -->
